packages/components/organisms/f-offers/src/components/Results.vue

```diff
@@ -0,0 +1,238 @@
+<template>
+    <div>
+        <content-cards
+            v-if="hasMounted"
+            :user-id="globalUserId"
+            :api-key="brazeApiKey"
+            :locale="$i18n.locale"
+            :push-to-data-layer="pushToDataLayer"
+            @has-loaded="hasLoaded"
+            @get-card-count="cardsReceived"
+            @on-error="onError">
+            <template #default="{ cards }">
+                <div :class="$style['c-offersResults-contentCards']">
+                    <template v-for="(card, i) in cards">
+                        <group-header-card
+                            v-if="card.type === 'Header_Card'"
+                            :key="i"
+                            :title="card.title" />
+                        <component
+                            :is="handleCustomCardType(card.type)"
+                            v-else
+                            :key="i"
+                            :card="card"
+                            :test-id="testIdForItemWithIndex(i)"
+                            :tenant="tenant"
+                        />
+                    </template>
+                </div>
+            </template>
+            <template #loading>
+                <skeleton-loader
+                    :type="loadingCard.type"
+                    :count="loadingCard.count" />
+            </template>
+        </content-cards>
+        <modal-copied-voucher-code
+            :is-open="isModalOpen"
+            :url="modalOngoingUrl"
+            :close-modal="closeModal" />
+    </div>
+</template>
+
+<script>
+import {
+    ContentCards,
+    PromotionCard,
+    VoucherCard,
+    GroupHeaderCard,
+    FirstTimeCustomerCard,
+    TermsAndConditionsCard,
+    SkeletonLoader
+} from '@justeat/f-content-cards';
+import '@justeat/f-content-cards/dist/f-content-cards.css';
+import { mapState } from 'vuex';
+import { VUEX_MODULE_NAMESPACE_OFFERS } from '../store/types';
+import ModalCopiedVoucherCode from './ModalCopiedVoucherCode.vue';
+
+export default {
+    name: 'OffersResults',
+
+    components: {
+        ContentCards,
+        PromotionCard,
+        VoucherCard,
+        GroupHeaderCard,
+        FirstTimeCustomerCard,
+        TermsAndConditionsCard,
+        SkeletonLoader,
+        ModalCopiedVoucherCode
+    },
+
+    data: () => ({
+        enabledCardTypes: [
+            'Header_Card',
+            'Promotion_Card_1',
+            'Promotion_Card_2',
+            'Recommendation_Card_1',
+            'Restaurant_FTC_Offer_Card',
+            'Voucher_Card_1',
+            'Anniversary_Card_1'
+        ],
+        title: 'Offers',
+        hasMounted: false,
+        contentCardsHaveLoaded: false,
+        error: undefined,
+        showContentCards: true,
+        tries: 0,
```

Comment:
`tries` doesn't have a purpose anymore if we're not attempting to reload on error.
```suggestion
```